CustomKnight/Skin/Base/Skinable/InvClaw.cs

```diff
@@ -0,0 +1,19 @@
+﻿
+namespace CustomKnight
+{
+    public class InvClaw : Skinable_Sprite
+    {
+        public static string Name = "Claw";
+        public InvClaw() : base(Name) { }
+        public override void SaveDefaultTexture()
+        {
+            GameObject cw = GameCameras.instance.hudCamera.gameObject.FindGameObjectInChildren("Inventory").FindGameObjectInChildren("Inv").FindGameObjectInChildren("Equipment").FindGameObjectInChildren("Mantis Claw");
+            ckTex.defaultSprite = CreateSpritewithppu(SpriteUtils.ExtractTextureFromSpriteLegacy(cw.GetComponent<SpriteRenderer>().sprite), cw.GetComponent<SpriteRenderer>().sprite.pixelsPerUnit); ;
+        }
+        public override void ApplySprite(Sprite sprite)
+        {
+            GameObject cw = GameCameras.instance.hudCamera.gameObject.FindGameObjectInChildren("Inventory").FindGameObjectInChildren("Inv").FindGameObjectInChildren("Equipment").FindGameObjectInChildren("Mantis Claw");
```

Comment:
Since a good number of the inventory objects need access to the Inv object or the Equipment object, we should probably cache them in skin manager.